certified-connectors/Kapish Power Connect/readme.md

```diff
@@ -0,0 +1,157 @@
+# Kapish Power Connect
```

Comment:
@BrettHankinson please include all required sections in the readme file. please see our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).